application/F3DOptionsTools.h

```diff
@@ -129,7 +129,8 @@ static inline const std::map<std::string_view, std::string_view> LibOptionsNames
   { "anti-aliasing", "render.effect.antialiasing.enable" },
   { "anti-aliasing-mode", "render.effect.antialiasing.mode" },
   { "tone-mapping", "render.effect.tone_mapping" },
-  { "final-shader", "render.effect.final_shader" },
+  { "final-shader", "render.effect.final_shader" }, 
+  { "textures-transform", "model.textures_transform" }
```

Comment:
```suggestion
  { "textures-transform", "model.textures_transform" },
```
keep trailing comma so this line isn't changed if/when one is added afterwards (also lets `clang-format` know not to merge lines I believe)